cmd/argo/commands/template/lint.go

```diff
@@ -24,6 +24,10 @@ func NewLintCommand() *cobra.Command {
 		Use:   "lint (DIRECTORY | FILE1 FILE2 FILE3...)",
 		Short: "validate a file or directory of workflow template manifests",
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
```

Comment:
The absence of the check is intentional. It allows you to use this this xargs

Reply:
I think xargs will be already accounted in args.

Reply:
no, I don't think so

```
find . -type f -name '*-workflow.yaml' | xargs argo lint
```

if find returns zero results - what happens?

Reply:
```
ls examples/aa*.yaml | xargs dist/argo template lint
ls: examples/aa*.yaml: No such file or directory
```

```
 ls examples/workflow-template/t*.yaml | xargs dist/argo template lint
WorkflowTemplate manifests validated
WorkflowTemplate manifests validated

```

Reply:
what about `true | xargs argo template lint`

Reply:
true | xargs dist/argo template lint
sbalasubram@INTUL1732f3e11 ~/go/src/github.com/argoproj/argo (issue-4285) $

Reply:
exit code != 0?

Reply:
true | xargs dist/argo template lint
sbalasubram@INTUL1732f3e11 ~/go/src/github.com/argoproj/argo (issue-4285) $ echo $?
0
it is returned existcode=0

Reply:
ok, do you want to add a test and we're good

Reply:
I will add e2e test